test/Conversion/amd/tritongpu_to_llvm_gfx1250.mlir

```diff
@@ -0,0 +1,28 @@
+// RUN:  triton-opt %s -split-input-file --allocate-shared-memory --convert-triton-amdgpu-to-llvm=arch="gfx1250" | FileCheck %s --check-prefix=GFX1250
+#linear = #ttg.linear<{register = [[0, 1], [0, 2], [0, 8], [0, 16], [0, 32], [0,64]], lane = [[1, 0], [2, 0], [4, 0], [8, 0], [0, 4]], warp = [[16, 0], [32, 0]], block = []}>
+#mma = #ttg.amd_wmma<{version = 3, warpsPerCTA = [4, 1], isTranspose = true, instrShape = [16, 16, 32]}>
```

Comment:
These 2 layouts have larger shapes than 32x32. Can you change the layout or use a larger shape?